sentry-android-integration-tests/sentry-uitest-android/src/main/java/io/sentry/uitest/android/ProfilingSampleActivity.kt

```diff
@@ -0,0 +1,109 @@
+package io.sentry.uitest.android
+
+import android.graphics.Bitmap
+import android.graphics.Color
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.ViewGroup
+import android.widget.ImageView
+import androidx.appcompat.app.AppCompatActivity
+import androidx.recyclerview.widget.LinearLayoutManager
+import androidx.recyclerview.widget.RecyclerView
+import androidx.test.espresso.idling.CountingIdlingResource
+import io.sentry.uitest.android.databinding.ActivityProfilingSampleBinding
+import io.sentry.uitest.android.databinding.ProfilingSampleItemListBinding
+import java.util.concurrent.ExecutorService
+import java.util.concurrent.Executors
+import kotlin.random.Random
+
+class ProfilingSampleActivity : AppCompatActivity() {
+
+    companion object {
+
+        /** The activity will set this when scrolling. */
+        val scrollingIdlingResource = CountingIdlingResource("sentry-uitest-android-profilingSampleActivityScrolling")
+    }
+
+    private lateinit var binding: ActivityProfilingSampleBinding
+    private val backgroundThreadPoolSize = 2
+    private val executor: ExecutorService = Executors.newFixedThreadPool(backgroundThreadPoolSize)
+    private var resumed = false
```

Comment:
`l`: Why do we need that property?

Reply:
it was a flag to stop the function after killing the activity, and i forgot to put it in fibonacci. Not a real issue, as the test is ran only once and the process is killed when the test ends. But i'm going to add it anyway